src/libPersistence/Retriever.cpp

```diff
@@ -152,11 +153,12 @@ bool Retriever::RetrieveTxBlocks(bool trimIncompletedBlocks) {
     }
   }
 
-  if (boost::filesystem::exists("StateDeltaFromS3")) {
+  if (boost::filesystem::exists(STORAGE_PATH + "/StateDeltaFromS3")) {
```

Comment:
Again I would recommend adding `const std::string STATE_DELTA_S3_PATH = "/StateDeltaFromS3";` to this file so the path can be easily updated at a later date.